src/util/taproot.rs

```diff
@@ -710,19 +729,21 @@ impl ControlBlock {
         Ok(self.size())
     }
 
-    /// Serialize the control block. This would be required when
-    /// using ControlBlock as a witness element while spending an output via
-    /// script path. This serialization does not include the VarInt prefix that would be
-    /// applied when encoding this element as a witness.
+    /// Serializes the control block.
+    ///
+    /// This would be required when using [`ControlBlock`] as a witness element while spending an
+    /// output via script path. This serialization does not include the [`::VarInt`] prefix that would
+    /// be applied when encoding this element as a witness.
     pub fn serialize(&self) -> Vec<u8> {
         let mut buf = Vec::with_capacity(self.size());
         self.encode(&mut buf).expect("writers don't error");
         buf
     }
 
-    /// Verify that a control block is correct proof for a given output key and script
-    /// This only checks that script is contained inside the taptree described by
-    /// output key, full verification must also execute the script with witness data
+    /// Verifies that a control block is correct proof for a given output key and script.
+    ///
+    /// Only checks that script is contained inside the taptree described by output key, full
```

Comment:
In f488093a04a02573507fcb8bdbd35b4bed7ba7b2:

This comma should probably be a full stop.

Reply:
Done, thanks.